src/server/req_manager.c

```diff
@@ -2790,115 +2785,84 @@ make_host_addresses_list(char *phost, u_long **pul)
 		}
 	}
 
-#if defined(__hpux)
-		hp = gethostbyname(phost);
-		if (hp == NULL) {
-			sprintf(log_buffer,
-					"addr not found for %s h_errno=%d errno=%d",
-					phost, h_errno, errno);
-			return (PBSE_UNKNODE);
-		}
+	memset(&hints, 0, sizeof(struct addrinfo));
+	/*
+		*      Why do we use AF_UNSPEC rather than AF_INET?  Some
+		*      implementations of getaddrinfo() will take an IPv6
+		*      address and map it to an IPv4 one if we ask for AF_INET
+		*      only.  We don't want that - we want only the addresses
+		*      that are genuinely, natively, IPv4 so we start with
+		*      AF_UNSPEC and filter ai_family below.
+		*/
```

Comment:
Looks like the rest of this comment was indented

Reply:
Thank Bhroam, fixing the indentation issue